src/client/components/ImageMenu.js

```diff
@@ -139,71 +166,61 @@ ImageMenu.propTypes = {
 }
 
 const getBorder = props => {
-  let color
-
-  if (props.isDragAccept) color = '#00e676'
-  else if (props.isDragReject) color = '#ff1744'
-  else if (props.isDragActive) color = '#2196f3'
-  else if (props.dragging) color = '#446fe9'
-  if (color) return `2px ${color} dashed`
-  else return '2px white solid'
+  if (props.isDragAccept) return `2px #00e676 dashed;`
+  if (props.isDragReject) return `2px #ff1744 dashed;`
+  if (props.isDragActive) return `2px #2196f3 dashed;`
+  if (props.dragging) return `2px #446fe9 dashed;`
+
+  return `1px solid rgba(209, 209, 209, ${getBackgroundAlpha(props)});`
 }
 
 const isVisible = props =>
   props.isDragAccept ||
   props.isDragReject ||
   props.isDragActive ||
-  props.dragging
+  props.dragging ||
+  props.active
 
-const getVisibility = props => {
-  if (isVisible(props)) return 'visible'
-  else return 'hidden'
-}
-const getOpacity = props => {
-  if (isVisible(props)) return '0.8'
-  else return '0'
-}
+const getVisibility = props => (isVisible(props) ? 'visible' : 'hidden')
+
+const getBackgroundAlpha = props => (isVisible(props) ? '0.8' : '0')
 
 const ImageMenuStyled = styled.div`
   .imageHover:hover & {
     visibility: visible;
-    transition: visibility 0.3s linear, opacity 0.3s linear;
-    visibility: visible;
-    opacity: 0.8;
+    transition: all 0.3s linear;
```

Comment:
I see we're adding this here, to `.imageHover:hover`, and we're adding it to the root `ImageMenuStyled` on line 198. Do we need both?

Reply:
Forsooth. Removed the surplus one.